docs/getting-started.md

```diff
@@ -254,23 +259,25 @@ To get an IPFS daemon working locally, do the following:
 
 If you get stuck, you can follow the instructions from the IPFS website [here](https://ipfs.io/docs/getting-started/).
 
-Once you've started the IPFS daemon you can run `yarn deploy` in the subgraph directory (make sure to pass the right host and port to 'graph deploy' via the '--ipfs' argument). `yarn deploy` should have been set up in section 3.3 when you went through the [Graph CLI documentation](https://github.com/graphprotocol/graph-cli). This will compile the mappings, and deploy the mappings, schema and the subgraph manifest itself to IPFS.
-
 To confirm the subgraph is stored on IPFS, you can pass that subgraph ID into `ipfs cat` to view the subgraph manifest with file paths replaced by IPLD links.
 
-### 4.2 Deploying The Subgraph
-
-Deploying the subgraph means the subgraph is running, and sourcing data from Ethereum, transforming that data with the mappings, and storing it in the Graph Node. Note that a running subgraph can safely be stopped and started, and it will pick up where it has left off. 
-
-#### 4.2.1 Create the Postgres db 
+#### 4.2 Create the Postgres db 
 
 Make sure you have Postgres installed. Then run the following commands:
 
 `initdb -D .postgres`
+`pg_ctl -D .postgres -l logfile start`
 `createdb <POSTGRES_DB_NAME>`
 
 Name the database something relevant to the project, so you always know how to access it. 
 
+### 4.3 Deploying The Subgraph
+
+Deploying the subgraph means that the Subgraph is ingesting all the events from the blockchain, transforming the data with mappings, and storing it in the Graph Node. Note that running a subgraph can safely be stopped and started, and it will pick up where it has left off. 
+
+Now you can run `yarn deploy --watch` in the subgraph directory (make sure to pass the right host and port to `graph deploy` via the '--ipfs' argument). `yarn deploy` should have been added to `package.json` in section 3.3 when you went through the [Graph CLI documentation](https://github.com/graphprotocol/graph-cli). This will compile the mappings, and deploy the mappings, schema and the subgraph manifest itself to IPFS.
```

Comment:
> in section 3.3 when you went through the [Graph CLI documentation](https://github.com/graphprotocol/graph-cli).

in section 3.3. This can be found in [Graph CLI documentation](https://github.com/graphprotocol/graph-cli).